README.md

```diff
@@ -104,7 +105,7 @@ _Note_: The foundation model can be found under their associated task.
 ### Tools
 
 - [Text Generation Inference](https://huggingface.co/docs/text-generation-inference/index): Toolkit to serve large language models.
-- [Text Embedding Inference](https://huggingface.co/docs/text-embeddings-inference/index): Toolkit to serve text embeddings.
+- [TGI Benchmark](https://www.youtube.com/watch?v=jlMAX2Oaht0): Understanding throughput of inference servers, with TGI example.
```

Comment:
Why remove TEI here?

Reply:
... I should not commit after midnight. I thought I duplicated it by inadvertance. Should be good now!